### ISSUE TITLE
Specify size of icon in toolbar left Element

### ISSUE THREAD

**Issue by AbdoulayeKarimTraore** [xotahal/react-native-material-ui#34]
I cannot specify the size of the icon by passing size in the props of the toolbar as follows
```                    
<Toolbar
    style={{ container: { elevation: 0 }, leftElement: {size: 30} }}
    leftElement='keyboard-arrow-left'
    centerElement={this._toolbarTitle()}
    rightElement={{
        actions: ['person-add', 'comment'],
        menu: { labels: ['Item 1', 'Item 2'] },
    }}
/>
```
the only way to change the size of the icon is when i change the global uiTheme spacing.iconSize, I looked in the code and you do take it into account in IconToggle but the passing of size in the leftElement props generates a failed prop type warning that states that size is an invalide key :), is this the correct way of changing the size of the icon or is there another way that I don't know about ? using version 1.3.2

**Comment by xotahal** [comment]
Hey @AbdoulayeKarimTraore ,

`size` prop is just sugar for `style={width: 24, height: 24}` - globally in this project. So you can always and everywhere use `width` and `height`.

But you are right, it won't work in this case, because there is a bug in `IconToogle` :/

**Comment by AbdoulayeKarimTraore** [comment]
Ok I understand, I love your module and I will do my utmost to try and help with the documentation as i'm getting to know it better

**Comment by xotahal** [comment]
Fixed, it works in 1.3.4 version ;)

<img width="396" alt="screen shot 2016-10-30 at 10 58 18 am" src="https://cloud.githubusercontent.com/assets/3531955/19835763/ca004dc2-9e8f-11e6-817a-4daaf941d514.png">

You can use `size`:
![screen shot 2016-10-30 at 10 58 41 am](https://cloud.githubusercontent.com/assets/3531955/19835773/e3268ffa-9e8f-11e6-9897-c63318d7c6e9.png)

Or `style`:
![screen shot 2016-10-30 at 10 58 49 am](https://cloud.githubusercontent.com/assets/3531955/19835774/e9f449d0-9e8f-11e6-9ce5-e7cab4f86694.png)
